pkg/gardenlet/controller/federatedseed/seed_control.go

```diff
@@ -29,12 +29,16 @@ import (
 	"github.com/gardener/gardener/pkg/gardenlet/apis/config"
 	confighelper "github.com/gardener/gardener/pkg/gardenlet/apis/config/helper"
 	"github.com/gardener/gardener/pkg/gardenlet/controller/federatedseed/extensions"
+	apiservernamespace "github.com/gardener/gardener/pkg/gardenlet/controller/federatedseed/network-policy"
```

Comment:
Can you name the package like the controller?

```suggestion
"github.com/gardener/gardener/pkg/gardenlet/controller/federatedseed/seedapiservernetworkpolicy"
```